### ISSUE TITLE
Numeric Feature Extraction

### ISSUE THREAD

**Issue by rdslater-tc** [RasaHQ/rasa_nlu#853]
**Rasa NLU version** (e.g. `0.7.3`): '0.12.0a1'

**Used backend / pipeline** (`mitie`, `spacy_sklearn`, ...): space_sklearn

**Operating system** (windows, osx, ...): osx

**Issue**: We cannot extract the numeric features of our training data.  
```python
from rasa_nlu.training_data import load_data
from rasa_nlu.config import RasaNLUConfig
from rasa_nlu.model import Trainer
from rasa_nlu.components import ComponentBuilder
builder = ComponentBuilder(use_cache=True)
training_data = load_data('./Code/va-ml/nlu_intent/data/processed/training_data.json')
trainer = Trainer(RasaNLUConfig("./config_spacy.json"))
trainer.train(training_data)
model_directory = trainer.persist('./projects/default/')
from rasa_nlu.model import Metadata, Interpreter
model_directory='./projects/default/default/model_20180215-130526/'
interpreter = Interpreter.load(model_directory, RasaNLUConfig("./config_spacy.json"),builder)
labels = [e.get("intent") for e in training_data.intent_examples]
set(labels)
[example.get("text_features") for example in training_data.intent_examples]
```
Last line returns "None" for all instances.  Previously this contained the numeric features from our pipeline. (~ver 0.10).  We pulled the last 3 lines straight from the GitHub SVC classifier and still are getting nothing for intent examples.  We do get a set of labels for our intents (I used set just to m are sure we could see all the classes rather than 1000 lines of our most popular. intent).

sourcefile: rasa_nlu/rasa_nlu/classifiers/sklearn_intent_classifier.py (line 94)

**Content of configuration file** (if used & relevant):
```json

```

**Comment by tmbo** [comment]
This is because the training data is not modified in place anymore but on a copy. So the training pipeline still adds that attribute to the examples, but you can not access it on this training data instance.

**Comment by tmbo** [comment]
You should be able to access the text features by processing your training samples with the loaded interpreter.

**Comment by rdslater-tc** [comment]
So there is no way currently to pull out the numeric-enconded training data?  We can get back our original text but we were interested in the numeric features to possibly test/write new classifiers.
